pkg/idp/provider/azure/azure.go

```diff
@@ -244,7 +247,8 @@ func (a *Azure) CreateOrUpdateSecret(id string, config provider.AppConfig, ctx c
 
 	// Create secret if it does not exist
 	if needsCreation {
-		secret, err = a.Client.ApplicationsById(id).AddPassword().Post(ctx, GetSecretCreateRequestBody(config), nil)
+		passwordRequestBody := applications.NewItemAddPasswordPostRequestBody()
```

Comment:
Does this work/get the same result as `GetSecretCreateRequestBody(config)`?

Reply:
I am not really sure, this is what I understood from what I saw in the library